transmogrifier/sources/ead.py

```diff
@@ -22,6 +22,13 @@ def get_optional_fields(self, xml: Tag) -> dict:
         """
         fields: dict = {}
 
+        if collection_description := xml.metadata.find("archdesc"):
```

Comment:
I was rethinking since `archdesc` is required and non-repeatable for an EAD record but I suppose we may use other sources that have EAD records for levels below `collection` so I'll add it back